x-pack/packetbeat/LICENSE-Npcap.txt

```diff
@@ -0,0 +1,228 @@
+

```

Comment:
Is this directory the right place for this or should it go in the root with additional words to note who it's relevant to — it probably already needs some of that to note that it's only relevant to Windows users.